spec/helpers/friends_helper_spec.rb

```diff
@@ -15,7 +13,7 @@
 
     it 'returns the last checkin address if it exists' do
       expect(helper.friends_device_last_checkin(device.checkins.first)).to match('Last available')
-      expect(helper.friends_device_last_checkin(device.checkins.first)).to match(device.checkins.first.fogged_city.to_s)
+      expect(helper.friends_device_last_checkin(device.checkins.first)).to match(device.checkins.first.city.to_s)
```

Comment:
No action except helper here. And it [operates](https://github.com/earlymarket/Coposition/blob/363f41dbd8e01585f7fb8a414c697d8aa29979de/app/helpers/friends_helper.rb#L4) with 'city' attribute @t-harps .